### DISCUSSION TITLE
Alternative to dropdowns  (migrating from Bootstrap to USWDS)

### DISCUSSION THREAD

**Discussion by jdonis** [uswds/uswds#5386] [Q&A]
Hello, 
I'm migrating from Bootstrap and my current code is using dropdowns buttons.
What would be the alternative?  

https://getbootstrap.com/docs/5.0/components/dropdowns/

Thks

**Comment by sarah-sch** [comment]
> **Community update — September 17, 2026:** The Slack guidance below is historical. For current questions and participation, use [USWDS GitHub Discussions](https://github.com/uswds/uswds/discussions). Past calls remain available in the [monthly call archive](https://designsystem.digital.gov/about/monthly-calls/).

Thank you for your question and sorry for the slow reply. [This discussion](https://gsa-tts.slack.com/archives/C3F14AHSQ/p1704376194535609) on the [#uswds-public Slack channel](https://designsystem.digital.gov/about/community/) contains some useful information.

Here's what one of our developers said on Slack:
_USWDS does not offer a tabs component at this time due to research needed surrounding accessibility and hidden content concerns.
We do offer an [Accordion](https://designsystem.digital.gov/components/accordion/) component which allows you to condense pages where appropriate.
We do offer dropdowns in the form of a [select](https://designsystem.digital.gov/components/select/) component, or [combobox](https://designsystem.digital.gov/components/combo-box/) component depending on your needs!_

**Comment by ryparker** [comment]
The Bootstrap component you linked is a button that opens a menu of links or actions. The earlier reply about tabs did not address that use case.

USWDS does not currently have a standalone, general-purpose equivalent to Bootstrap's dropdown button. Choose the replacement based on the contents:

- **Site navigation links:** use the appropriate [header navigation](https://designsystem.digital.gov/components/header/) pattern.
- **A small group of actions:** a [button group](https://designsystem.digital.gov/components/button-group/) can keep the actions visible.
- **Choosing a form value:** use [select](https://designsystem.digital.gov/components/select/) or [combo box](https://designsystem.digital.gov/components/combo-box/). These are not substitutes for an action menu.

If the design specifically needs a collapsed action menu, it needs a custom implementation with the appropriate keyboard and focus behavior. Styling a button with USWDS does not supply that behavior automatically.
